### DISCUSSION TITLE
Open Telemetry traces for SqlClient are not getting exported to Console

### DISCUSSION THREAD

**Discussion by Trivedi-Ashish** [open-telemetry/opentelemetry-dotnet#3418] [Q&A]
Open Telemetry traces for SqlClient are not getting exported to Console when I am using locally built sqlClient 4.1.0 DLL.
I have a sample application (.NET 6 Console app). I am connecting to local Sql server database from the application.
For debugging purposes, I am using SqlClient 4.1.0 locally built DLL.
I am able to debug the sqlClient without any issue but I am not able to see OpenTelemetry traces from SqlClient. I am exporting the traces to console.

If I use SqlClient 4.1.0 as a nuget package, then I am able to see the OpenTelemetry traces.

Package used for OpenTelemetry tracing:
1. OpenTelemetry 1.3.0
2. OpenTelemetry.Instrumentation.SqlClient
3. OpenTelemetry.Exporter.Console 1.3.0

Locally built DLLs:
Microsoft.Data.SqlClient 4.1.0

**Comment by cijothomas** [comment]
Not sure how much help we can offer, since the official SQLClient nuget is working, but your private local build is not.
If you share a minimal repro, we can take a look.
